<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordAddLinkDialog</class>
 <widget class="QDialog" name="RecordAddLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>666</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QWidget" name="dialogContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>743</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>776</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#dialogContainer {
	border-radius: 12px;
	background-color: white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QWidget" name="headerRow" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="styleSheet">
            <string notr="true">color: black;
font-family: Open Sans;
font-weight: 500;
font-size: 22px;
line-height: 30px;
letter-spacing: 0%;
</string>
           </property>
           <property name="text">
            <string>Add Links</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="stepsGroup" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border-radius: 4px;
	background-color: #D9D9D9;
}

QPushButton:checked {
	border-radius: 4px;
	background-color: #22084B;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="step1Check">
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>8</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>8</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="step2Check">
              <property name="minimumSize">
               <size>
                <width>128</width>
                <height>8</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>8</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../Resources/Resources.qrc">
             <normaloff>:/images/Project/cross-icon.svg</normaloff>:/images/Project/cross-icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="step1Group" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#summaryGroup {
	background-color:transparent;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QLineEdit" name="searchLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background: white;
	font-family: 'Open Sans';
	font-size: 14px;
	padding-left: 6px;
	padding-right: 12px;
	border: 1px solid #34343426;
	border-radius: 6px;
	border-width: 1px;
	spacing:10px;
	color: #808080;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
}</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Search record here</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="recordTableView">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>228</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>228</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QTableView {
	color: #4D4D4D;
    border: 1px solid #eee;
    border-radius: 8px;
    background-color: white;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
}

QHeaderView::section {
	font-family: Open Sans;
	font-weight: 600;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
    background-color: #fafafa;
    padding: 8px;
    border: none;
    font-weight: 600;
    color: #333;
}

QTableView::item {
    padding: 12px;
    border-bottom: 1px solid #f1f1f1;
}

QTableView::item:selected {
    background-color: transparent;
}

QScrollBar {
    background: transparent;
}

QScrollBar:vertical {
    width: 6px;
    background: transparent;
}
QScrollBar::handle:vertical {
    background: #4B286D;
    border-radius: 3px;
}
QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0;
}

QTableView::indicator:unchecked {
    width: 20px;
    height: 20px;
    border: 2px solid #555;
    border-radius: 4px;
    background: white;
}

QTableView::indicator:checked {
    width: 20px;
    height: 20px;
    border: 0px solid #22084B;
    border-radius: 4px;
    background-color: white;
	image: url(:/images/Record/check-icon.svg);
}</string>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="step2Group" native="true">
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid #cccccc;
    border-radius: 6px;
    padding: 8px 32px 8px 12px;
    font-size: 14px;
    color: #333;
    background-color: #ffffff;
}

QComboBox:focus {
    border: 2px solid #9a6aff; /* Purple focus border */
}

QComboBox:editable {
    background: white;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border: none;
}

QComboBox::down-arrow {
	image: url(:/images/chevron-down.svg);
    width: 16px;
    height: 16px;
}

QComboBox::down-arrow:on {
    image: url(:/images/chevron-up.svg);
}

/* Popup dropdown */
QComboBox QAbstractItemView {
    border: 1px solid #dddddd;
    border-radius: 10px;
    padding: 6px;
    background-color: #ffffff;
    selection-background-color: #f2f2f2;
    outline: none;
    font-size: 14px;
    color: #333333;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>18</number>
         </property>
         <item>
          <widget class="QWidget" name="relationshipFieldGroup" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="relationshipFieldLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 16px;
line-height: 24px;
letter-spacing: 0%;
</string>
              </property>
              <property name="text">
               <string>Relationship</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="relationshipFieldCombo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Select relationship here</string>
              </property>
              <item>
               <property name="text">
                <string>Up</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Down</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="linkTypeFieldGroup" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="linkTypeFieldLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 16px;
line-height: 24px;
letter-spacing: 0%;
</string>
              </property>
              <property name="text">
               <string>Link Type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="linkTypeFieldCombo">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Select link type here</string>
              </property>
              <item>
               <property name="text">
                <string>Link</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Realizes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Allocates</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Satisfies</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottomGroup" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1.5px solid #1A1A1A
}
</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: #FCCA69;
	color: white;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
}
</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordAddLinkDialog</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordAddLinkDialog</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordAddLinkDialog</receiver>
   <slot>onNextButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCloseButtonClicked()</slot>
  <slot>onNextButtonClicked()</slot>
  <slot>onSearchChanged(QString)</slot>
 </slots>
</ui>
